include/s3select_oper.h

```diff
@@ -802,7 +802,9 @@ class value
   
   bool operator!=(const value& v)
```

Comment:
bool operator!=(const value& v)
is getting NULL 
but return true/false result 
while it should return UNKNOWN